Doc/library/asyncio-eventloop.rst

```diff
@@ -361,7 +361,7 @@ Creating Futures and Tasks
 
    .. versionadded:: 3.5.2
 
-.. method:: loop.create_task(coro, *, name=None, context=None)
+.. method:: loop.create_task(coro, *, name=None, context=None, eager_start=...)
```

Comment:
Why not `eager_start=None` like the others?

Reply:
```suggestion
.. method:: loop.create_task(coro, *, name=None, context=None, eager_start=None)
```